controllers/authController.js

```diff
@@ -0,0 +1,60 @@
+import jwt from 'jsonwebtoken';
+import AppError from '../errors/AppError';
+import { save, getUserByUsername, comparePassword } from '../models/UserModel';
+
+const logger = require('../utils/logger')('authController');
+
+
+const register = async (req, res, next) => {
+    logger.log('debug', 'register: %j', req.body);
+    const { body } = req;
+    try {
+        const user = await save({
+            username: body.username,
+            email: body.email,
+            hashedPassword: body.hashedPassword,
+            name: body.name,
+            surname: body.surname,
+            dateOfBirth: body.dateOfBirth
+        });
+        logger.log('info', `Successfully registered: ${body.username}`);
+        res.status(200).send({ payload: { user: user } });
+    } catch (error) {
+        next(new AppError(error.message, 400));
+    }
+};
+
+const login = async (req, res, next) => {
+    logger.log('debug', 'logIn: %j', req.body);
+    try {
+        const user = await getUserByUsername(req.body.username);
+        if (!user) {
+            logger.log('debug', 'Login failed');
```

Comment:
change level do error and message to something more informative like at what username login failed so we could get some statistics